x-pack/plugin/transform/qa/multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/transform/integration/TransformIntegTestCase.java

```diff
@@ -149,6 +149,14 @@ protected StartTransformResponse startTransformWithRetryOnConflict(String id, Re
             try (RestHighLevelClient restClient = new TestRestHighLevelClient()) {
                 return restClient.transform().startTransform(new StartTransformRequest(id), options);
             } catch (ElasticsearchStatusException e) {
+                logger.warn(
```

Comment:
Should we log the whole exception's stack trace by supplying `e` as an argument to `warn`?

Reply:
I decided against that because `toString()` lacks important information, e.g. it misses the status code and other details. 

Note that after 10 retries, we get a stack trace.

Reply:
Ok, got it.